plugins/labhub.py

```diff
@@ -279,14 +279,18 @@ def difficulty_level(user, iss):
                 2. The user belongs to developers or maintainers team as well as
                    newcomers team.
                 False if
-                1. A newcomer asks for assignment to an issue that has no
+                1. A member asks for assignment to an issue that has
+                   initiatives/gci label.
+                2. A newcomer asks for assignment to an issue that has no
                    difficulty label.
```

Comment:
E127 continuation line over-indented for visual indent

*Origin: PycodestyleBear (E127), Section: `all.autopep8`.*